### ISSUE TITLE
Support for WPF usercontrol placed within MFC application

### ISSUE THREAD

**Issue by sheejaGY** [pywinauto/pywinauto#685]
I am trying to automate an MFC application. In that one WPF usercontrol is embedded with in CWnd.
In that case how can I access WPF usercontrol and other wpf controls inside this usercontrol.
Does pywinauto support this?

**Comment by airelil** [comment]
In short, pywinauto should be able to manipulate any controls that are visible through Inspect utility. Have a short read at our docs: https://pywinauto.readthedocs.io/en/latest/getting_started.html?highlight=Inspect#gui-objects-inspection-spy-tools, if you hadn't. Notice that Inspect tool has two different modes, and some elements can be accessed only through 'UI automation' or through 'MSAA' mode.

**Comment by vasily-v-ryabov** [comment]
Yes, Getting Started Guide explains how to try different backends. Probably you've tried default `Application(backend="win32")` but WPF doesn't support Win32 API related stuff, you have to use `Application(backend="uia")` to deal with WPF controls. It should work perfectly if "UI Automation" features are not disabled on app side.

**Comment by santim07** [comment]
Will it support tray application. I have an application which is displayed at try but with no name only icon. can i open that.

**Comment by vasily-v-ryabov** [comment]
@santim07 yes, if this icon is in the toolbar. Which Windows version are you using?

**Comment by santim07** [comment]
Windows 10

**Comment by santim07** [comment]
Windows 10

**Comment by vasily-v-ryabov** [comment]
I think we can try to write some example. I don't have enough time to do that right now, will mark as a documentation issue which means "TODO: add an example". Current `test_taskbar.py` tests work with Win7, Win8.1 and WinServer SysTray.

**Comment by santim07** [comment]
Thanks
